<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-04-05T09:34:40 IST" name="Suite" finished-at="2023-04-05T09:34:51 IST" duration-ms="11125">
    <groups>
    </groups>
    <test started-at="2023-04-05T09:34:40 IST" name="Test" finished-at="2023-04-05T09:34:51 IST" duration-ms="11125">
      <class name="testCases.loginTest">
        <test-method is-config="true" signature="beforetest()[pri:0, instance:testCases.loginTest@3cae7b8b]" started-at="2023-04-05T09:34:40 IST" name="beforetest" finished-at="2023-04-05T09:34:47 IST" duration-ms="6916" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforetest -->
        <test-method signature="login()[pri:1, instance:testCases.loginTest@3cae7b8b]" started-at="2023-04-05T09:34:47 IST" name="login" finished-at="2023-04-05T09:34:50 IST" duration-ms="2819" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Login Failed expected [http://main.stage.bt-ms.co] but found [https://ecompartner.digitalmesh.co.in/]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Login Failed expected [http://main.stage.bt-ms.co] but found [https://ecompartner.digitalmesh.co.in/]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at pages.loginPage.loginClick(loginPage.java:46)
at testCases.loginTest.login(loginTest.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="editProfile()[pri:2, instance:testCases.loginTest@3cae7b8b]" started-at="2023-04-05T09:34:51 IST" name="editProfile" finished-at="2023-04-05T09:34:51 IST" duration-ms="8" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- editProfile -->
        <test-method signature="logoutuser()[pri:3, instance:testCases.loginTest@3cae7b8b]" started-at="2023-04-05T09:34:51 IST" name="logoutuser" finished-at="2023-04-05T09:34:51 IST" duration-ms="246" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutuser -->
        <test-method signature="userReg()[pri:4, instance:testCases.loginTest@3cae7b8b]" started-at="2023-04-05T09:34:51 IST" name="userReg" finished-at="2023-04-05T09:34:51 IST" duration-ms="8" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userReg -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:testCases.loginTest@3cae7b8b]" started-at="2023-04-05T09:34:51 IST" name="afterTest" finished-at="2023-04-05T09:34:51 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- testCases.loginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
